docs/src/pages/api-reference/server.mdx

```diff
@@ -110,6 +112,11 @@ downloaded instead of previewed in the browser. You can read more about content
 disposition on
 [MDN ↗](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition).
 
+<Callout>
+  ACL can only be overridden if set on the UploadThing dashboard. Attempts to
+  upload a mismatching ACL setting will error.
```

Comment:
It is a little unclear what this means, I assume "if you don't allow overriding ACLs on a per-request basis from the dashboard, attempting to set one here will result in an error", but I am not 100% sure.

Reply:
Yea if you have public set and havent toggled the "Allow overriding", attemptikg to upload with private acl will error.

Same for duration on getSignedURL